packages/common-ui-web/test/components/TakePictureButton.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TakePictureButton } from '../../src';
+
+const TAKE_PICTURE_BTN_TEST_ID = 'take-picture-btn';
+const TAKE_PICTURE_OUTER_BTN_TEST_ID = 'take-picture-outer-btn';
+
+describe('TakePictureButton component', () => {
+  it('should take the size prop into account', () => {
+    const size = 60;
+    const borderWidth = 3;
+    const margin = 3;
+    const { unmount } = render(<TakePictureButton disabled={false} size={size} />);
+
+    const outerDiv = screen.getByTestId(TAKE_PICTURE_OUTER_BTN_TEST_ID);
+    const innerDiv = screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID);
+    expect(outerDiv.style.width).toEqual(`${size - (2 * borderWidth)}px`);
+    expect(outerDiv.style.height).toEqual(`${size - (2 * borderWidth)}px`);
+    expect(innerDiv.style.width).toEqual(`${size - (2 * borderWidth) - (2 * margin)}px`);
+    expect(innerDiv.style.height).toEqual(`${size - (2 * borderWidth) - (2 * margin)}px`);
+    unmount();
+  });
```

Comment:
- Update this test with the dynamic sizes that I mentioned above
- Only pass the necessary props (you don't care about the disabled prop)
- Add a test for the default value

```typescript jsx
function expectSize(size: number): void {
  const expectedInnerSize = size * 0.84;
  const expectedBorderRadius = size * 0.16 / 4;
  const expectedOuterSize = size - 2 * expectedBorderRadius;

  const outerLayer = screen.getByTestId(OUTER_LAYER_TEST_ID);
  const innerLayer = screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID);

  expect(outerDiv.style.width).toEqual(`${expectedOuterSize}px`);
  expect(outerDiv.style.height).toEqual(`${expectedOuterSize}px`);
  expect(outerDiv.style.borderWidth).toEqual(`${expectedBorderRadius}px`);
  expect(innerDiv.style.width).toEqual(`${expectedInnerSize}px`);
  expect(innerDiv.style.height).toEqual(`${expectedInnerSize}px`);
  expect(innerDiv.style. margin).toEqual(`${expectedBorderRadius}px`);
}

it('should take the size prop into account', () => {
  const size = 100;
  const { unmount } = render(<TakePictureButton size={size} />);

  expectSize(size);
  unmount();
});

it('should use a size of 60 by default', () => {
  const { unmount } = render(<TakePictureButton />);

  expectSize(60);
  unmount();
});
```